src/main/java/com/suse/saltstack/netapi/calls/modules/Pkg.java

```diff
@@ -188,4 +188,26 @@ private Pkg() { }
         return new LocalCall<>("pkg.info_available", Optional.of(Arrays.asList(packages)),
                 Optional.empty(), new TypeToken<Map<String, Info>>(){});
     }
+
+    public static LocalCall<Boolean> upgradeAvailable(String packageName) {
+        LinkedHashMap<String, Object> args = new LinkedHashMap<>();
+        args.put("package", packageName);
+        return new LocalCall<>("pkg.upgrade_available", Optional.empty(), Optional.of(args),
+                new TypeToken<Boolean>(){});
+    }
+
+    public static LocalCall<String> latestVersion(String packageName) {
+        LinkedHashMap<String, Object> args = new LinkedHashMap<>();
+        args.put("package", packageName);
+        return new LocalCall<>("pkg.latest_version", Optional.empty(), Optional.of(args),
+                new TypeToken<String>(){});
```

Comment:
The [documentation] (https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.zypper.html#salt.modules.zypper.latest_version) doesn't actually mention that there is a `package` keyword argument. Did that change since this PR was opened or is it just not correctly documented?